.lintstagedrc.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Any reason not to store the lintsage config in package.json that's how we used to have this set up in the past.

Reply:
Just personal preference of being able to see where different configs live, I'm happy to move it into the package.json if you'd prefer?

Reply:
I think that's fine, let's keep it this way since we're setting up husky hooks outside of package.json as well.